src/Core/src/Layouts/Flex.cs

```diff
@@ -990,27 +990,10 @@ public void init(Item item, float width, float height)
 				ordered_indices = null;
 				if (item.ShouldOrderChildren && item.Count > 0)
 				{
-					var indices = new int[item.Count];
```

Comment:
An alternate idea would be to use `ArrayPool<int>` here, as it's a temporary array.